conferences/models.py

```diff
@@ -1,18 +1,20 @@
 from django.db import models
 from django.utils.translation import ugettext as _
+from django.core.exceptions import ValidationError
 
 
 class Place(models.Model):
-    place_name = models.CharField(
+    name = models.CharField(
         max_length=300
     )
-    place_url = models.CharField(
-        max_length=300
+    url = models.CharField(
+        max_length=300,
+        blank=True
```

Comment:
Zamiast  `CharField` możesz użyć `URLField` - załatwia to problem walidacji czy pole zawiera poprawną wartość dla urla